### PR TITLE
Feature/snapmail implementation

### PR DESCRIPTION
**Pull request by farhantariq12b**

### CODE REVIEW

File: store/snapmail.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  },
+  setSnapmailById(state, payload) {
+    state.snapmailById = payload;
+  },
+}
+
+export const actions = {
+  async createSnapmail({}, payload) {
+    try {
+      await this.$axios.$post(`${process.env.API_URL}/snapmails`, payload);
```
Comment (JaviCT) [inline store/snapmail.js]: Remove the  `/` before snapmails for all http request, already included in the environment variable

Comment (farhantariq12b) [reply store/snapmail.js]: @JaviCT sure, I'll remove it. But I think its a more standard practise to put the `/` here. If that is not easy in terms of change like `env var` on server, .env files for developer etc.

Comment (JaviCT) [reply store/snapmail.js]: @farhantariq12b Yes, you are right but it is already done without `/` in all project. I will create an issue to fix it in the future.

Comment (farhantariq12b) [reply store/snapmail.js]: ok sure, got it.

**Pull request merged**